tests/js_api/src/api.ts

```diff
@@ -77,9 +85,14 @@ export function createApi(): Promise<ApiPromise> {
                 commitment: 'KateCommitment',
                 appLookup: 'DataLookup'
             },
+            V2HeaderExtension: {
+                commitment: 'KateCommitmentV2',
+                appLookup: 'DataLookup'
+            },
             HeaderExtension: {
                 _enum: {
                     V1: 'V1HeaderExtension',
+                    V2: 'V2HeaderExtension',
```

Comment:
Are we dropping this for the testnet?

Reply:
I am waiting for @kroos47 and @MiguelDD1 advice on that